nni/contrib/compression/base/wrapper.py

```diff
@@ -125,6 +164,18 @@ def unwrap(self):
 
         self.module.forward = self.module_forward
         delattr(self.module, '_nni_wrapper')
+        # delete bias which is created during module fusion process
+        if getattr(self.module, "original_bias", None) is not None:
+            delattr(self.module, 'bias')
+            original_bias = self.module.original_bias
+            if isinstance(original_bias, nn.parameter.Parameter):
+                self.module.register_parameter('bias', torch.nn.Parameter(original_bias.detach().clone()))
+            elif isinstance(original_bias, torch.Tensor):
+                self.module.register_buffer('bias', original_bias.detach().clone())
+            delattr(self.module, 'original_bias')
+        if len(self.fused_modules) > 0 and not self.is_bias and check_bias(self.module):
+            delattr(self.module, 'bias')
+            self.module.register_parameter('bias', None)
```

Comment:
what if the original module don't have a None bias?

Reply:
I mean that some module like `nn.Softmax` do not have a parameter bias?
maybe `check_bias` should have three states: 'Tensor', 'None', 'non-exist'?